tests/components/vodafone_station/test_config_flow.py

```diff
@@ -216,3 +209,31 @@ async def test_reauth_not_successful(hass: HomeAssistant, side_effect, error) ->
 
         assert result2["type"] == FlowResultType.ABORT
         assert result2["reason"] == "reauth_successful"
+
+
+async def test_options_flow(hass: HomeAssistant) -> None:
+    """Test options flow."""
+
+    mock_config = MockConfigEntry(domain=DOMAIN, data=MOCK_USER_DATA)
+    mock_config.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.vodafone_station.config_flow.VodafoneStationSercommApi.login",
+    ), patch(
+        "homeassistant.components.vodafone_station.config_flow.VodafoneStationSercommApi.logout",
+    ), patch(
+        "homeassistant.components.vodafone_station.async_setup_entry",
+    ):
```

Comment:
Please be reminded that currently when you submit the options flow, nothing will happen. If you want to apply the new options directly, you should reload the integration and add proper listeners to that.

In that light I don't want to see the async_setup_entry patched, but rather executed and then tested if we reload them

Reply:
Fixed! Will update `fritz` as well once this get merged.

Reply:
This is not fixed as far as I can see